src/Cache/FileHandler.php

```diff
@@ -26,6 +26,8 @@
  */
 final class FileHandler implements FileHandlerInterface
```

Comment:
@rcwsr , to be able to continue with he PR, I need a test case that is failig without changes from #8996 and working with them, so we can ensure this won't get accidentally broken.

Reply:
I put an example case of 30MB sample cache file, it loads the cache properly

#9027